src/setup-ghidra-source.cmake

```diff
@@ -13,6 +14,14 @@ set_property(CACHE sleigh_GHIDRA_RELEASE_TYPE PROPERTY STRINGS "stable" "HEAD")
 set(ghidra_version "10.1.4")
 set(ghidra_git_tag "Ghidra_10.1.4_build")
 set(ghidra_shallow TRUE)
+
+set(additional_patches)
+
+# TODO(Ian): do something better with previously applied patch
+foreach(PFILE ${sleigh_ADDITIONAL_PATCHES})
+  set(additional_patches ${additional_patches} COMMAND patch -p0 -i "${PFILE}" || true)
+endforeach()
```

Comment:
Gotcha this was originally patch because I was generating patch files with https://pypi.org/project/diff-match-patch/ which are in the gnu and not git format. We now shell out to diff anyways so I should be able to shell out to git instead to generate git patches

Reply:
For context, patches are currently generated here: https://github.com/lifting-bits/remill/blob/40ade70f86e28f5d06b0e7f1bcd528ced70b6034/scripts/replace_pc_relative_disactions/replace_pc_relative_disactions/main.py